docker/Dockerfile_windows_amd64

```diff
@@ -0,0 +1,15 @@
+FROM --platform=linux/amd64 ubuntu:jammy-20231211.1
```

Comment:
I think we have other Dockerfiles like this one. Let's make sure they're all using the same ubuntu version?

Reply:
Can't, unfortunately.

The other Ubuntu image doesn't have a new enough version of `mingw-w64` to be able to cross compile for Windows.

This Ubuntu image has `glibc` version 2.35, which is too high for cross compiling Linux arm64 without a breaking change.